internal/pkg/template/templates/workloads/partials/cf/imported-alb-resources.yml

```diff
@@ -0,0 +1,142 @@
+# For one listener or the port 443 listener if two.
+{{- $arn := ""}}
```

Comment:
Why not the arn for always http listener? If we only have one listener that's the one, otherwise it'll be a http listener redirecting to HTTPS. This way it is guaranteed that this arn is always for http listener.

Or if the logic is still complicated we can probably split it into two different files for these two scenarios.

Reply:
I was trying not to duplicate the code for the `forward` rule--either it's the only rule or it's one of the two rules. Splitting it would make the logic easier to follow for sure, but may be overkill? Let me move some logic out of the template and see if it's clearer.